### ISSUE TITLE
Drones not attacking sometimes 

### ISSUE THREAD

**Issue by heyheyhey123243** [TeamPneumatic/pnc-repressurized#669]
### Minecraft Version
1.12
### Forge Version
14.23.5.2854
### Mod Version
pneumaticcraft-repressurized-1.12.2-0.11.15-398.jar
### Describe your problem, including steps to reproduce it
I have attack drones in my world with the following program https://imgur.com/a/4Z4o99v (Not sure if this is the most optimal program, all I want it to do is attack a mob that spawns in a specific area, then go to a charging station to refuel and pick up ammo if necessary. It seems to work)

I have 6 drones with that program covering an area of 70 x 25 and another 6 covering a different area of 70 x 25.

If I spawn a mob in the first of those areas, all 6 will go to attack and it all works like normal. However, if I go to the second area where the other set of 6 drones are at and spawn a mob, they will just sit there and do nothing. If I wrench and replace them in the second area drones, they will function but then the first area with the drones will stop functioning until I replace them as well.

TLDR: Drones will just not attack in some areas until I replace them with the wrench.

I also have this other problem where the drone will go to the chest to pick up the item but then will idle near the chest instead of the charging station. While it's doing this and I spawn a mob, it will not go and attack that mob, instead, it just sits near the chest doing nothing.

**Comment by desht** [comment]
Sorry, 1.12 is no longer supported.

**Comment by heyheyhey123243** [comment]
Commenting again because I have a fix and just in case anyone in the future runs into a similar problem.

The issue isn't with pneumaticcraft itself but with optifine and how it chunkloads entities. 

https://www.reddit.com/r/feedthebeast/comments/5fqezo/pneumaticcraft_drones_they_seem_to_stop_working/

Fix: 
1. Uninstall optifine or
2. Disable smooth world loading and lazy chunk loading
